scripts/google/covidharvard/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pandas
```

Comment:
Can we centralize in https://github.com/datacommonsorg/data/blob/master/requirements.txt ?